### ISSUE TITLE
Sessions after a canceled session are rejected automatically

### ISSUE THREAD

**Issue by yarcub** [onsip/SIP.js#751]
**Describe the bug**
All the invites after a canceled one are rejected automatically (`sip.js@0.15.6`).

Canceled sessions don't seem to update the Simple instance status like they do on termination.
The snippet that rejects the invite based on the `state`.
https://github.com/onsip/SIP.js/blob/ed5a6ce738379eb9c79aa9c41b1b1cf716b893d4/src/Web/Simple.ts#L132

**Browser Logs**
```
Mon Oct 28 2019 14:43:48 GMT+0000 (Western European Standard Time) | sip.invite-dialog | INVITE dialog a3f03e4dcd3d635d7a7c067bac9214443421-pivotcallvb10715mvnje5QevHe63Z1j constructed
Mon Oct 28 2019 14:43:55 GMT+0000 (Western European Standard Time) | sip.inviteservercontext | closing INVITE session a3f03e4dcd3d635d7a7c067bac9214443421-pivotcallje5QevHe63Z1j
Mon Oct 28 2019 14:43:56 GMT+0000 (Western European Standard Time) | sip.invite-dialog | INVITE dialog a3f03e4dcd3d635d7a7c067bac9214443421-pivotcallvb10715mvnje5QevHe63Z1j destroyed
Mon Oct 28 2019 14:44:13 GMT+0000 (Western European Standard Time) | sip.invite-dialog | INVITE dialog 498c646f28e60771c1b45992481122487578-pivotcallev3vqs5m3opjaUN8mvt7rcH constructed
Mon Oct 28 2019 14:44:13 GMT+0000 (Western European Standard Time) | sip.simple | Rejecting incoming call. Simple only supports 1 call at a time
Mon Oct 28 2019 14:44:13 GMT+0000 (Western European Standard Time) | sip.inviteservercontext | rejecting RTCSession
Mon Oct 28 2019 14:44:13 GMT+0000 (Western European Standard Time) | sip.inviteservercontext | closing INVITE session 498c646f28e60771c1b45992481122487578-pivotcallpjaUN8mvt7rcH
Mon Oct 28 2019 14:44:13 GMT+0000 (Western European Standard Time) | sip.invite-dialog | INVITE dialog 498c646f28e60771c1b45992481122487578-pivotcallev3vqs5m3opjaUN8mvt7rcH destroyed
```
**SIP MESSAGES**
https://gist.github.com/yarcub/54168155690b40642048f78b1fe82b07

**To Reproduce (if possible)**
1.  Receive an invite
2. Cancel the invite
3. Receive an invite

**Expected behavior**
The `ringing` event handler would be called.
https://sipjs.com/api/0.15.0/simple/#ringing

**Observed behavior**
The invite is automatically rejected

**Environment Information**
 - Chrome 78.0.3904.70

**Additional context**

**Comment by egreenmachine** [comment]
@john-e-riordan is currently updating Simple. We will make sure this is covered.

**Comment by john-e-riordan** [comment]
I've confirmed the port of `Simple` to `SimpleUser` which is built on the new API code does not have this issue. `SimpleUser` is currently in #757 along with demos which use it and associated ongoing work.

**Comment by egreenmachine** [comment]
This is fixed in the new `SimpleUser` of `0.15.7`.
